servicepulse/intro-archived-messages.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Archived Message Management in ServicePulse
+summary: Describes how ServicePulse monitors archived failed messages, and allows un-archiving archived failed messages
+tags:
+- ServicePulse
+reviewed:  
+---
+
+### Introduction
+
+ServiceControl version 1.12.0 and higher has a configurable [ErrorRetentionPeriod](/servicecontrol/creating-config-file.md) that schedules the disposal of archived failed messages. In SerivcePulse 1.5.0 and higher screens to manage archived messages have been introduced.
+
+Archived Messages can be found by going to the failed messages screen.
+
+![Archived Messages Tab](images/archive.png)
+
+### Archived Messages
+
+The archived message page will open showing messages archive in the last 2 hours. Users can select from the available predefined ranges to limit the set of messages you are reviewing
```

Comment:
> archive 

archived?

> Users can select from the available predefined ranges to limit the set of messages you are reviewing

To limit the set of displayed messages, select an option from the available predefined ranges